swagger.json

```diff
@@ -5796,6 +5861,44 @@
             "items": {
                 "$ref": "#/definitions/DocumentType"
             }
+        },
+        "Contact": {
+            "type": "object",
+            "description": "Information about a contact.",
+            "properties": {
+                "id": {
+                    "type": "integer",
+                    "description": "ID of the contact",
+                    "format": "int64",
+                    "example": 123
+                },
+                "userId": {
```

Comment:
What is the `userId` in our system? Contacts don't have references to our `users` table, do they?

Reply:
`contacts` does in fact have a FK to `users`

Reply:
Oh, interesting. The Model struct has a foreign key annotation. Weird.